app/assets/stylesheets/application/notifications.scss

```diff
@@ -159,4 +159,8 @@ h1#clear_notifcations {
       margin-bottom: 5px;
     }
   }
+}
+
+.reset_notification{
```

Comment:
It would also be cool to rename this to something more descriptive of what it actually does, but I'm a little at a loss since I already suggested `dim` for the other class. It would be nice to only have one opacity change (either 0.8 or 0.5), but that's more of a design question so... maybe something about having the notification modal open..?